Add server daemonset yaml template files

sufuf3 opened the pull request:
- Add tcp & unix with namespace vortex
- Fake image name

---

- [x] tcp can run
- [x] unix can run

test files are under `~/go/src/github.com/linkernetworks/network-controller/example/server-yaml`

sufuf3 commented:
I added the following config in these yaml files.
```yaml
        securityContext:
          privileged: true
```

Could you please help me review this PR, again?
Many thanks.

John-Lin commented on deploy/server/unix-daemonset.yaml:
```diff
@@ -0,0 +1,39 @@
+        volumeMounts:
+        - mountPath: /var/run/docker.sock
+          name: docker-sock
+        - mountPath: /tmp/a.sock
```
rename to `/var/run/vortex.sock`

John-Lin commented on deploy/server/unix-daemonset.yaml:
```diff
@@ -0,0 +1,39 @@
+          path: /var/run/docker.sock
+      - name: grpc-sock
+        hostPath:
+          path: /tmp/a.sock
```
rename to /var/run/vortex.sock

John-Lin commented on deploy/server/unix-daemonset.yaml:
```diff
@@ -0,0 +1,39 @@
+        securityContext:
+          privileged: true
+        command: ["./network-controller-server"]
+        args: ["-unix=/tmp/a.sock"]
```
rename to `/var/run/vortex.sock`

John-Lin commented on deploy/server/tcp-daemonset.yaml:
```diff
@@ -0,0 +1,34 @@
+metadata:
+  name: network-controller-server-tcp
+  namespace: vortex
+  labels:
```
At this moment. we don't need this label

John-Lin commented on deploy/server/unix-daemonset.yaml:
```diff
@@ -0,0 +1,39 @@
+metadata:
+  name: network-controller-server-unix
+  namespace: vortex
+  labels:
```
At this moment. we don't need this label

sufuf3 commented:
I removed labels. Also, updated the socket name. 
Mount path need to be setup folder, so I use `/var/run` as `volumes.hostPath.path` & `volumeMounts.mountPath`.

Please help me review this PR, again.
Thank you very much.

John-Lin commented on deploy/server/unix-daemonset.yaml:
```diff
@@ -0,0 +1,37 @@
+          path: /var/run/docker.sock
+      - name: grpc-sock
+        hostPath:
+          path: /var/run/
```
I found mount entire `/var/run/` can cause significant security issue. we should change back to `/tmp`

John-Lin replied:
Unix sockets only live while the program is running, so `/tmp/` is usually an alright place for them to live
